packages/xo-server-test/sample.config.toml

```diff
@@ -3,6 +3,13 @@
   email = ''
   password = ''
 
+[servers]
+  [servers.default]
+    username = ''
+    password = ''
+    host = ''
+    allowUnauthorized = true
+
```

Comment:
`autoConnect` default value is `true`

Reply:
I think we have to let `autoConnect = false` because if we add a server, we try to connect with `server.enable` just after it. We will know if it is the addition or the connection of the server which failed (if it fails). What do you think ?

Reply:
Good idea :+1:

Reply:
I just understand what you meant in your first comment. You were right. Sorry.